SimonKaya91/Weather-Server
# Adding routes

**TESTING**

* Pull branch down, make sure tests run and are outputting correctly, ensure that when you change the assertion in the tests the tests fail or pass depending on what you try.

* When you pull down the branch run ```rm -rf node_modules/ && npm it```

* Check that all dependencies we have installed are being used if not comment on here and we can look together.

* Make sure that npm start still works and that navigating to a route still works, wont return anything though due to nothing in the one route we have. Paste after npm start in browser ```localhost:3000/```
